components/02-components/02-tab/tab.hbs

```diff
@@ -1,10 +1,10 @@
  <div class="tab-wrapper">
      <div class="tab-header">
         <h2 class="h3 blue mb-4">{{group-title}}</h2>
-         <ul class="nav nav-tabs">
+         <ul class="nav nav-tabs" role="tablist">
              {{#each tabs}}
                 {{#if @first}}
-                <li><a class="active" data-bs-toggle="tab" href="#{{id}}">{{title}}</a></li>
+                <li role="presentation"><a class="active" data-bs-toggle="tab" href="#{{id}}">{{title}}</a></li>
```

Comment:
should the presentation role be attached to every li or even the parent ul?

Reply:
Yep, you're right! I missed the other li's. I went off of this syntax:
https://getbootstrap.com/docs/5.2/components/navs-tabs/#javascript-behavior

Technically, the `a` tag has the `role="tab"`` put in dynamically, so the `role="presentation"` helps ignore the `li`.